src/Command/BaseCacheCommand.php

```diff
@@ -14,38 +14,26 @@
 namespace Sonata\CacheBundle\Command;
 
 use Sonata\Cache\CacheManagerInterface;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command;
 
-/**
- * NEXT_MAJOR: stop extending ContainerAwareCommand.
- */
-abstract class BaseCacheCommand extends ContainerAwareCommand
+abstract class BaseCacheCommand extends Command
```

Comment:
This is a BC break, but the previous implementation didn't work because ContainerAwareCommand does not exist on Symfony 5